Doc/library/mailcap.rst

```diff
@@ -60,6 +60,18 @@ standard.  However, mailcap files are supported on most Unix systems.
    use) to determine whether or not the mailcap line applies.  :func:`findmatch`
    will automatically check such conditions and skip the entry if the check fails.
 
+   .. versionchanged:: 3.11
```

Comment:
update with the 3.10.x version number for this branch.

Reply:
```suggestion
   .. versionchanged:: 3.10.6
```